VERSION

```diff
@@ -1 +1 @@
-0.5.12
+0.6.0-nordix
```

Comment:
Can you also tag it to make it easier to control version using mix `tag: "..."`?

Reply:
Of course! We'll tag it when this is merged. I though this was already merged and tagged, but it was forgotten.

Reply:
Hey guys, I believe we can call it just 0.6.0 from now on. I'm confident we will get `eredis_cluster` ownership and I doubt there will be any new versions from adrienmo/eredis_cluster.

Reply:
I don't know what this VERSION file is for, but it's updated to 6.0.0 now.